### PR TITLE
Fix/open gov pagination

### PR DESCRIPTION
**Pull request by This-Is-Prince**

### CODE REVIEW

File: pages/member-referenda/index.tsx [hunk 1]
```diff
@@ -14,40 +15,95 @@ import { networkTrackInfo } from '~src/global/post_trackInfo';
...
+				page,
+				proposalType,
+				sortBy,
+				trackNo: trackId
+			});
+		} else {
+			prev[currTrackName] = getOnChainPostsCount({
+				network,
+				page,
+				proposalType,
```
Comment (KanishkaRajputd) [inline pages/member-referenda/index.tsx:65]: trackStatus is missing here

File: src/components/Listing/FellowshipReferendum/TrackListingTabContent.tsx [hunk 2]
```diff
@@ -22,72 +16,51 @@ const GovernanceCard = dynamic(() => import('~src/components/GovernanceCard'), {
 
 interface ITrackListingTabContentProps {
 	className?: string;
-	trackName: string;
```
Comment (KanishkaRajputd) [inline src/components/Listing/FellowshipReferendum/TrackListingTabContent.tsx:25]: trackName can be number also

File: src/components/Listing/Tracks/TrackListingStatusTabContent.tsx [hunk 3]
```diff
@@ -20,44 +20,48 @@ interface ITrackListingStatusTabContentProps {
 	trackName: string;
 	posts: any[];
```
Comment (KanishkaRajputd) [inline src/components/Listing/Tracks/TrackListingStatusTabContent.tsx:21]: track name can be number also

File: pages/member-referenda/index.tsx [hunk 4]
```diff
@@ -14,40 +15,95 @@ import { networkTrackInfo } from '~src/global/post_trackInfo';
...
 const FellowshipAdmin: FC<IFellowshipReferendumProps> = (props) => {
-	const { data, error, network } = props;
+	const { posts, error, network } = props;
 	const { setNetwork } = useNetworkContext();
 	setNetwork(network);
 
 	if (error) return <ErrorState errorMessage={error} />;
-	if (!data) return <PostEmptyState />;
-	const { posts } = data;
-	console.log('posts', posts);
```
[the remark this answers is not in the document]
Comment (sahas-01) [reply pages/member-referenda/index.tsx:50]: Done, fixed

File: pages/member-referenda/index.tsx (hunk 1 above)
Comment (sahas-01) [reply pages/member-referenda/index.tsx:65]: Added

**Pull request merged**